consolidator/modules/kafkahealthcheck/kafkahealthcheck.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/jballerina.java;
+import consolidatorService.config;
+import ballerina/log;
+
+public function getAllTopicsKafka() returns handle|error? {
+    handle bootStrapServer = java:fromString(config:KAFKA_BOOTSTRAP_NODE);
+    handle newMosipKafkaAdminClientResult = newMosipKafkaAdminClient(bootStrapServer);
+    log:printInfo("Checking if metadata Topics are present");
+    handle|error? result = trap getAllTopics(newMosipKafkaAdminClientResult);
```

Comment:
In our case we will have many topics, is there any other lite way of checking ?

Reply:
done

Reply:
resolved